.gitmodules

```diff
@@ -243,13 +243,18 @@
 [submodule "third_party/silabs/gecko_sdk"]
 	path = third_party/silabs/gecko_sdk
 	url = https://github.com/SiliconLabs/gecko_sdk.git
-	branch = v4.2.3
+	branch = v4.3.2
 	platforms = efr32
 [submodule "third_party/silabs/wiseconnect-wifi-bt-sdk"]
 	path = third_party/silabs/wiseconnect-wifi-bt-sdk
 	url = https://github.com/SiliconLabs/wiseconnect-wifi-bt-sdk.git
-	branch = 2.7.1-Matter
+	branch = 2.8.2
 	platforms = efr32,silabs_docker
+[submodule "third_party/silabs/wifi_sdk"]
+	path = third_party/silabs/wifi_sdk
+	url = https://github.com/SiliconLabs/wiseconnect.git
+	branch = v3.1.0
+	platforms = efr32
```

Comment:
shoudn't it be `platforms = efr32,silabs_docker` too now if you added it to the docker

Reply:
actually, you don't want to list `silabs_docker` everything that has `silabs_docker` will be cloned in the container, since this one is already present in it, no need to clone it.

Reply:
We are not using it anyway as BRD4338A is not supported. For `wiseconnect-wifi-bt-sdk` I have done the changes in the #29261 since this CI still expects the repo to be cloned and changes can be brought in when we change to this docker (version 15 -> 17)

Reply:
`silabs_docker` is used to indicate the items we want to clone inside the docker using the `checkout_submodules.py`, and since this will be preinstalled on the docker we don't need to clone it. Let me know if my understanding is incorrect.

Reply:
Ah right